src/ring_theory/algebra_tower.lean

```diff
@@ -4,10 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Kenny Lau
 -/
 
+import algebra.algebra.tower
 import algebra.invertible
-import ring_theory.adjoin
 import linear_algebra.basis
-import algebra.algebra.basic
+import ring_theory.adjoin
 
 /-!
 # Towers of algebras
```

Comment:
Is the docstring below here still accurate, or have the things it describes moved file?

Reply:
It is still accurate.